WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        [TestMethod]
+        public async Task Should_Get_User_By_Id_Async()
+        {
+            // Setup

```

Comment:
I think we didn't define it in our guidelines, but we usually use `Arrange `(instead of setup) - `Act `- `Assert `blocks for tests.